installer/kfserving/description.yaml

```diff
@@ -3,8 +3,8 @@ description: |
   Serverless Inferencing on Kubernetes
 namespace: kfserving-system
 install:
-  - type: manifest
-    location: kfserving-v0.5.1-v1beta1.yaml
+  - type: kustomize
+    location: https://github.com/fuseml/extensions/installer/kfserving/kustomize
```

Comment:
Is there anything that we customize in these files, or can we get them from their original upstream location instead of maintaining them in our github repo ?

Reply:
I'm not sure, it almost looks like @flaviodsr actually patched upstream files ... https://github.com/fuseml/fuseml/commit/9dfc7884e481e0629bb88713804d92845033e758

Reply:
The reason for using kustomize was exactly to use the upstream kfserving yaml without any changes.

Unfortunately kustomize only supports referencing a resource from a URL starting from version `3.5.5` which means it would require kubectl 1.21+. So, that is the reason we have the `base/kfserving.yaml` file here.